test/e2e/install/install_test.go

```diff
@@ -86,9 +86,13 @@ func getTestClusterDetails() (string, int, []string, error) {
 	// Redundant check, but keep it anyway, we are writing a test after all.
 	// We should have found at least one node!
 	if len(nodeIPs) == 0 {
-		return "", 0, nil, errors.New("no usable nodes found")
+		return "", "", 0, nil, errors.New("no usable nodes found")
 	}
 
+	tag := os.Getenv("e2e_image_tag")
```

Comment:
hmm.. follow up to readability/maintainability above -- I see ... well... Also golang's CamelCase doesn't help. well... do as you see fit.

Reply:
and more importantly it should be uppercase. But then we should change all e2e variables.

Reply:
Indeed. As discussed above constants should be uppercase (which these are/should be), variables (in the shell) should not. But it's and agreement/style, not really meaning too much and one can use any case for shell variables (For me consistency across layers is much more important and that can be solved by `-i` to grep and code-search tools often have "case insensitive" search).